### ISSUE TITLE
Card Container

### ISSUE THREAD

**Issue by asanhix** [chingu-voyage3/grad.then#57]
Card container is the direct parent of cards.
* Do we need the container or a separator component will be enough?
* If no what would be its behavior and data?

**Comment by mar-bi** [comment]
This component is a tricky one.
Here I list all its forms that are presented in our app

1. **4-Cards Container** 
 
![4-cards-container](https://user-images.githubusercontent.com/25195616/34043106-404b6cbc-e1a8-11e7-9509-23f0a63fc545.png)

2. **5-Cards Container**

![5-cards-container](https://user-images.githubusercontent.com/25195616/34043141-5e915786-e1a8-11e7-82e1-fb491f71223c.png)

3. **2-Cards Container with text**
*this one can be simplified as **"3-Cards Container"***

![2-cards-and-text-container](https://user-images.githubusercontent.com/25195616/34043158-70d7afc6-e1a8-11e7-8e58-53405390d3e5.png)

4. **2-`Story Card` Container - 2 rows**

![2-cards-container-2rows](https://user-images.githubusercontent.com/25195616/34043198-9f65f410-e1a8-11e7-96ad-cc49f303e885.png)

5. **10-Cards Container - 2 rows**
*basically it is **"5-Cards Container"***
![10-cards-container](https://user-images.githubusercontent.com/25195616/34043212-b45aaee2-e1a8-11e7-8645-bce5e49a4dad.png)

Here we can see some pattern:
- 2-Cards (columns) container (with 2 rows)
- 3-Cards (columns) container
- 4-Cards (columns) container
- 5-Cards container (columns) (containers with 1 and 2 rows)

The number of rows doesn't matter as we use **CSS grid**.
I'd like to try to make it a single component which receives the **number of columns** via one of its `props`. :thinking: :nerd_face:

**Comment by asanhix** [comment]
@mar-bi can we use media-queries here. As viewport width changes our Cards container  can decide to how may columns go into  a row?

**Comment by mar-bi** [comment]
not exactly `@media` but awesome features of **css grid**.
I'd like to play&build this component. :baby_chick: :smile:

**Comment by asanhix** [comment]
:+1:

**Comment by mar-bi** [comment]
The `CardContainer` is going to look like this
1. `LEARN

![screenshot-2017-12-20 grad then](https://user-images.githubusercontent.com/25195616/34210409-683a3510-e59e-11e7-89b5-5a298befc064.png)

2. `EVENTS`

![screenshot-2017-12-20 grad then 1](https://user-images.githubusercontent.com/25195616/34210542-da885b74-e59e-11e7-8556-6a89d124f236.png)

3. `SUPPORT`

![screenshot-2017-12-20 grad then 2](https://user-images.githubusercontent.com/25195616/34210549-e8bd70d0-e59e-11e7-828d-c1d68367ed49.png)

4. `STORIES`

![screenshot-2017-12-20 grad then 3](https://user-images.githubusercontent.com/25195616/34210579-023a82b4-e59f-11e7-8e09-db44c2f0be5c.png)

**Comment by spences10** [comment]
Sweet! 🎊 🎉
